change/@ni-nimble-components-1fb65473-c803-4f85-ad46-02ad52a5b22b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add missing anchored region import to menu component",
```

Should we mention the Storybook 7 upgrade for the changelog?

```suggestion
  "comment": "Upgrade from Storybook 6.5 to Storybook 7.0. Add missing anchored region import to menu component",
```

---

My thinking in not mentioning it was that it doesn't affect the contents of this package. It's more of an infrastructure change that clients will get when they visit our docs whether they uptake the new package version or not. But there's no harm in mentioning it, if only to remind ourselves of the change when we get ready to do demos next iteration.